modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -40,324 +40,467 @@
-    // [STRING] "cdf", "json" or "osfls"
-    constexpr const char* KeyInputFileType = "InputFileType";
-    // [STRING] should be path to folder containing the input files
-    constexpr const char* KeySourceFolder = "SourceFolder";
-
-    // ---------------------- MANDATORY INPUT TYPE SPECIFIC KEYS ---------------------- //
-    // [STRING] Path to a .txt file containing seed points
-    constexpr const char* KeyCdfSeedPointFile = "SeedPointFile";
-    // [STRING] Currently supports: "batsrus", "enlil" & "pfss"
-    constexpr const char* KeyJsonSimulationModel = "SimulationModel";
-
-    // ----------------------- OPTIONAL INPUT TYPE SPECIFIC KEYS ---------------------- //
-    // [STRING ARRAY]
-    constexpr const char* KeyCdfExtraVariables = "ExtraVariables";
-    // [STRING]
-    constexpr const char* KeyCdfTracingVariable = "TracingVariable";
-    // [STRING]
-    constexpr const char* KeyJsonScalingFactor = "ScaleToMeters";
-    // [BOOLEAN] If value False => Load in initializing step and store in RAM
-    constexpr const char* KeyOslfsLoadAtRuntime = "LoadAtRuntime";
-
-    // ---------------------------- OPTIONAL MODFILE KEYS  ---------------------------- //
-    // [STRING ARRAY] Values should be paths to .txt files
-    constexpr const char* KeyColorTablePaths = "ColorTablePaths";
-    // [VEC2 ARRAY] Values should be entered as {X, Y}, where X & Y are numbers
-    constexpr const char* KeyColorTableRanges = "ColorTableRanges";
-    // [VEC2 ARRAY] Values should be entered as {X, Y}, where X & Y are numbers
-    constexpr const char* KeyMaskingRanges = "MaskingRanges";
-    // [STRING] Value should be path to folder where states are saved (JSON/CDF input
-    // => osfls output & oslfs input => JSON output)
-    constexpr const char* KeyOutputFolder = "OutputFolder";
-
-    // ------------- POSSIBLE STRING VALUES FOR CORRESPONDING MODFILE KEY ------------- //
-    constexpr const char* ValueInputFileTypeCdf = "cdf";
-    constexpr const char* ValueInputFileTypeJson = "json";
-    constexpr const char* ValueInputFileTypeOsfls = "osfls";
-
-    // --------------------------------- Property Info -------------------------------- //
     constexpr openspace::properties::Property::PropertyInfo ColorMethodInfo = {
-        "colorMethod",
+        "ColorMethod",
         "Color Method",
         "Color lines uniformly or using color tables based on extra quantities like, for "
         "examples, temperature or particle density."
     };
     constexpr openspace::properties::Property::PropertyInfo ColorQuantityInfo = {
-        "colorQuantity",
+        "ColorQuantity",
         "Quantity to Color By",
         "Quantity used to color lines if the 'By Quantity' color method is selected."
     };
-    constexpr openspace::properties::Property::PropertyInfo ColorQuantityMinInfo = {
-        "colorQuantityMin",
+    constexpr openspace::properties::Property::PropertyInfo ColorMinMaxInfo = {
+        "ColorQuantityMin",
```

Comment:
This identifier and name needs to be updated to match that it is now both the min and max and not just the min